Tests/Tests/Core/Extensions/UIKit/UIViewTests.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  UIViewTests.swift
+//  PovioKit_Tests
+//
+//  Created by Borut Tomazin on 19/09/2022.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import XCTest
+import UIKit
+@testable import PovioKit
+
+class UIViewTests: XCTestCase {
+  private let shadowPath: UIBezierPath = .init(rect: .init(x: 0, y: 0, width: 100, height: 100))
+  private let shadowColor: UIColor = .red
+  private let shadowRadius: CGFloat = 50
+  private let shadowOpacity: Float = 0.5
+  private let shadowOffset: CGSize = .init(width: 5, height: 10)
+  
+  func test_dropShadows_allParameters() {
+    let view = UIView()
+    view.layer.shadowColor = UIColor.blue.cgColor
+    XCTAssertNil(view.layer.shadowPath)
+    XCTAssertEqual(view.layer.shadowColor, UIColor.blue.cgColor)
+    XCTAssertEqual(view.layer.shadowOffset, .init(width: 0, height: -3))
+    XCTAssertEqual(view.layer.shadowOpacity, 0)
+    XCTAssertEqual(view.layer.shadowRadius, 3)
+    
+    view.dropShadow(path: shadowPath, shadowColor: shadowColor, radius: shadowRadius, opacity: shadowOpacity, offset: shadowOffset)
+    XCTAssertEqual(view.layer.shadowPath, shadowPath.cgPath)
+    XCTAssertEqual(view.layer.shadowColor, shadowColor.cgColor)
+    XCTAssertEqual(view.layer.shadowOffset, shadowOffset)
+    XCTAssertEqual(view.layer.shadowOpacity, shadowOpacity)
+    XCTAssertEqual(view.layer.shadowRadius, shadowRadius)
+  }
+  
+  func test_dropShadows_missingPath() {
+    let view = UIView()
+    XCTAssertNil(view.layer.shadowPath)
+    
+    view.dropShadow(path: nil, shadowColor: shadowColor, radius: shadowRadius, opacity: shadowOpacity, offset: shadowOffset)
+    XCTAssertNil(view.layer.shadowPath)
+  }
+  
+  func test_applyBorder() {
+    let color: UIColor = .blue
+    let width: CGFloat = 10
+    
+    let view = UIView()
+    view.layer.borderColor = UIColor.black.cgColor
+    XCTAssertEqual(view.layer.borderColor, UIColor.black.cgColor)
+    XCTAssertEqual(view.layer.borderWidth, 0)
```

Comment:
This feels like we are testing the UIKit framework instead of our extension, maybe we should make the assumption that this always works, except if we have business logic that relies on this.

If you agree with this, I would suggest removing such assertions in other places too.

Reply:
That's like a precondition so we know for sure how was before and how it is now.